app/connector/kafka/src/main/java/io/syndesis/connector/kafka/KafkaVerifierExtension.java

```diff
@@ -65,27 +62,68 @@ protected Result verifyConnectivity(Map<String, Object> parameters) {
     }
 
     private static void verifyCredentials(ResultBuilder builder, Map<String, Object> parameters) {
-        final String brokers = ConnectorOptions.extractOption(parameters, "brokers");
-        final String certificate = ConnectorOptions.extractOption(parameters, "brokerCertificate");
-        final String transportProtocol = ConnectorOptions.extractOption(parameters, "transportProtocol");
-        LOG.debug("Validating Kafka connection to {} with protocol {}", brokers, transportProtocol);
+        final String brokers = ConnectorOptions.extractOption(parameters, KafkaBrokerService.BROKERS);
+        final String certificate = ConnectorOptions.extractOption(parameters, KafkaBrokerService.BROKER_CERTIFICATE);
+        final String securityProtocol = ConnectorOptions.extractOption(parameters, KafkaBrokerService.SECURITY_PROTOCOL);
+        final String saslMechanism = ConnectorOptions.extractOption(parameters, KafkaBrokerService.SASL_MECHANISM);
+        final String saslLoginCallbackHandlerClass = ConnectorOptions.extractOption(parameters, KafkaBrokerService.SASL_LOGIN_CALLBACK_HANDLER_CLASS);
+        final String username = ConnectorOptions.extractOption(parameters, KafkaBrokerService.USERNAME);
+        final String password = ConnectorOptions.extractOption(parameters, KafkaBrokerService.PASSWORD);
+        final String oauthTokenEndpointURI = ConnectorOptions.extractOption(parameters, KafkaBrokerService.OAUTH_TOKEN_ENDPOINT_URI);
         if (ObjectHelper.isNotEmpty(brokers)) {
-            KafkaBrokerService kafkaBrokerService = new KafkaBrokerServiceImpl(brokers, transportProtocol, certificate);
-            try {
-                kafkaBrokerService.ping();
-            } catch (KafkaBrokerServiceException e) {
+            boolean validParameters = true;
+            if (KafkaBrokerService.SASL_SSL.equals(securityProtocol)) {
+                if (StringUtils.isBlank(username)) {
```

Comment:
We can avoid depending on Commons Lang by doing this

```suggestion
                if (ObjectHelper.isEmpty(username)) {
```

Reply:
Thanks for the tip, I will change it.